src/sonic-config-engine/config_samples.py

```diff
@@ -35,22 +35,22 @@ def generate_t1_sample_config(data):
 
 def generate_empty_config(data):
     new_data = {'DEVICE_METADATA': data['DEVICE_METADATA']}
-    if not new_data['DEVICE_METADATA']['localhost'].has_key('hostname'):
```

Comment:
is this necessary? https://www.tutorialspoint.com/python3/dictionary_has_key.htm

I can see python3 still have this support.

Reply:
indeed. Not sure why 2to3 converted. I used `2to3 -f all` switch. Are there recommended switches to use?

Reply:
removed in 3.1:

https://docs.python.org/3.1/whatsnew/3.0.html

Reply:
Can confirm, `has_key()` has been removed in Python 3.